Crane/app/src/androidTest/java/androidx/compose/samples/crane/details/DetailsActivityTest.kt

```diff
@@ -58,12 +58,20 @@ class DetailsActivityTest {
 
     @get:Rule(order = 1)
     val composeTestRule = AndroidComposeTestRule(
-        ActivityScenarioRule<DetailsActivity>(
+        activityRule = ActivityScenarioRule<DetailsActivity>(
             createDetailsActivityIntent(
                 InstrumentationRegistry.getInstrumentation().targetContext,
                 testExploreModel
             )
-        )
+        ),
+        activityProvider = { rule ->
```

Comment:
Can you add a comment about why is this needed? I'd expect this is not necessary to check

Reply:
This is currently being discussed in https://issuetracker.google.com/issues/174472899

Reply:
Can you add the link as a comment there? Some people might be wondering what's going on :)